src/verify-request/tests/verify-request.test.ts

```diff
@@ -43,6 +40,8 @@ describe('verifyRequest', () => {
       session.accessToken = 'test_token';
       session.scope = 'test_scope';
       await Shopify.Utils.storeSession(session);
+      
+      Shopify.Clients.Rest.prototype.get = jest.fn(() => Promise.resolve({ "body": "" } as RequestReturn));
```

Comment:
Can we add a comment here to indicate what we're expecting to mock?

We could also do something like this to make sure that no other calls are being made:
```suggestion
      Shopify.Clients.Rest.prototype.get = jest.fn(({path}) => {
        expect(path).toEqual('metafields');
        return Promise.resolve({ "body": "" } as RequestReturn;
      }));
```